analyzer/codechecker_analyzer/analyzers/analyzer_types.py

```diff
@@ -148,8 +149,11 @@ def check_supported_analyzers(analyzers):
 
     enabled_analyzers = set()
     failed_analyzers = set()
-
     for analyzer_name in analyzers:
+        analyzer_name, analyzer_version = analyzer_name.split('=', 1) \
```

Comment:
```suggestion
        analyzer_name, requested_version = analyzer_name.split('=', 1) \
```

Does this sound any better? Not sure, just an idea.

Reply:
Thanks for the suggestion! I fixed the variable name.